VirtoCommerce.Storefront.Model/Common/MutablePagedList.cs

```diff
@@ -238,14 +238,19 @@ public ICollection Keys
         {
             get
             {
-                throw new NotImplementedException();
+               TryReloadPagedData();
```

Comment:
The method 'TryReloadPagedData' tells us that there are only 2 ways of the results: true and false.
But we are using it like a void result. It's not the correct name of this method.